newapi/ooniapi/probe_services.py

```diff
@@ -46,55 +53,89 @@ def check_in():
             probe_cc:
               type: string
               description: Two letter, uppercase country code
+              example: IT
             probe_asn:
               type: string
+              description: ASN, two uppercase letters followed by number
+              example: AS1234
             platform:
               type: string
+              example: android
             software_version:
```

Comment:
I would add software_name like in the other APIs

Reply:
Done